packages/web/src/components/cells/hover.tsx

```diff
@@ -35,9 +35,11 @@ export function tsHover(
       end: end,
       create: () => {
         const tooltipContainer = document.createElement('div');
-        tooltipContainer.className = 'p-2 space-y-2 max-w-3xl max-h-96 overflow-scroll';
+        tooltipContainer.style.display = 'none';
```

Comment:
Are we able to use tailwind for this property too?

Reply:
hidden or invisible might work similarly but they're not the same as none I can try a swap in if you want

Reply:
`hidden` translates to `display: none`.

<img width="444" alt="Screenshot 2024-09-06 at 12 37 50 PM" src="https://github.com/user-attachments/assets/d103d104-abce-4576-a25e-90868315e7ab">

Reply:
thought hidden had some extra on it since it breaks with flex, maybe none breaks with flex, I'll swap it